touchforms/formplayer/static/formplayer/script/test_helpers/jasmine-fixture.js

```diff
@@ -0,0 +1,433 @@
+/* jasmine-fixture - 1.2.2
```

Comment:
i believe we gzip if the client supports it but don't automatically compress unless explicitly put in a `{% compress %}` templatetag.